README.md

```diff
@@ -1,75 +1,93 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Doctors Appointment Backend
```

Comment:
- [ ] Your project does not have a license file.  Kindly add a license file and section on your readme.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template/blob/master/README.md), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in a new "License" section of your readme.

That way, your app will be well documented 💪